vapi/vcenter/vcenter_vmtx.go

```diff
@@ -46,10 +46,51 @@ type Template struct {
 	VMHomeStorage        *DiskStorage          `json:"vm_home_storage,omitempty"`
 }
 
+// CPU defines Cores and CPU count
+type CPU struct {
```

Comment:
question: do we need to export these?

Reply:
Yes, we need to. This function can be used with the vcenter manager outside the package and to access the fields it should be exported.